roles/openstack_helm_designate/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2022 VEXXHOST, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Generate OpenStack-Helm endpoints
+  ansible.builtin.include_role:
+    name: openstack_helm_endpoints
+  vars:
+    openstack_helm_endpoints_repo_name: "{{ openstack_helm_designate_chart_repo_name }}"
+    openstack_helm_endpoints_repo_url: "{{ openstack_helm_designate_chart_repo_url }}"
+    openstack_helm_endpoints_chart: "{{ openstack_helm_designate_chart_name }}"
+
+- name: Deploy Helm chart
+  kubernetes.core.helm:
+    name: "{{ openstack_helm_designate_chart_name }}"
+    chart_ref: "{{ openstack_helm_designate_chart_repo_name }}/{{ openstack_helm_designate_chart_name }}"
+    chart_version: 0.2.7
+    release_namespace: openstack
+    kubeconfig: /etc/kubernetes/admin.conf
+    values: "{{ _openstack_helm_designate_values | combine(openstack_helm_designate_values, recursive=True) }}"
```

Comment:
With the change to Flux CD, things look a little different, see the following:

https://github.com/vexxhost/atmosphere/blob/d9e15a340da407775a1ce04877a7c85871dd72cb/roles/openstack_helm_barbican/tasks/main.yml#L23-L76